hw-learning-goal-label-key.md

```diff
@@ -0,0 +1,39 @@
+# HW Learning Goal Label Key
+This document describes the labels we give some of our homework assignments. These labels should help a student understand what the learning goal of the assignment.
+
+Often, we will assign homework in the following or similar structure:
+
+```
+3. Read about writing pseudocode: https://github.com/Ada-Developers-Academy/ [R]
+```
+
+The first portion is an explanation of the assignment and its link. The last portion is a label corresponding to our intended learning goal of the reading.
+
+The labels map to the following goals:
+
+Label | Learning Goal
+:--- | :---
+R | Reinforcement
+S | Supplementing, adding, or extending Context
+N | New material that will be immediately relevant
+
+Here is a brief description of each label:
+
+## Reinforcement [R]
+
+An assignment with the Reinforcement [R] label indicates that the goal of the reading is to go over a concept that has already been introduced into the curriculum.
+
+## Supplementing, Adding, or Extending Context [S]
+
+When an assignment has this label, it serves to add additional context to familiar concepts. Materials and assignments with this label will likely increase and round-out your understanding of a topic. It gives either a different way of looking at a concept, explains a concept more holistically, or shares a way that the concept is applied in a practical way.
+
+## New Material and Immediately Relevant [N]
+
+Reading with this label usually indicates that it is introducing a new concept and new material. If we are assigning new material as reading homework, it will become immediately relevant, likely the very next day.
+
```

Comment:
These are the ones we consider to be mandatory, right? Can we make that explicit?

Reply:
>These are the materials we consider to be the most important to our curriculum. If you read nothing else, read the [N]s.